docs/rules/require-description.md

```diff
@@ -39,13 +39,42 @@ type someTypeName {
 
 The schema defines the following properties:
 
-### `on` (array)
+### `types` (boolean)
 
-The object is an array with all elements of the type `string`.
+Includes:
 
-Additional restrictions:
+- `ObjectTypeDefinition`
+- `InterfaceTypeDefinition`
+- `EnumTypeDefinition`
+- `ScalarTypeDefinition`
+- `InputObjectTypeDefinition`
+- `UnionTypeDefinition`
 
-* Minimum items: `1`
+### `overrides` (object)
+
+Configuration for precise `ASTNode`
+
+Properties of the `overrides` object:
+
+#### `ObjectTypeDefinition` (boolean)
```

Comment:
My guess is that the definitions for each of these won't be self-evident for most folks. It may be worth including either a basic definition of what each is or a link to their reference in graphql spec:

https://spec.graphql.org/June2018/#ObjectTypeDefinition
https://spec.graphql.org/June2018/#InterfaceTypeDefinition
https://spec.graphql.org/June2018/#EnumTypeDefinition
https://spec.graphql.org/June2018/#ScalarTypeDefinition
https://spec.graphql.org/June2018/#InputObjectTypeDefinition
https://spec.graphql.org/June2018/#UnionTypeDefinition
https://spec.graphql.org/June2018/#FieldDefinition
https://spec.graphql.org/June2018/#InputValueDefinition
https://spec.graphql.org/June2018/#EnumValueDefinition
https://spec.graphql.org/June2018/#DirectiveDefinition

Reply:
@JustinTRoss I like your proposal, how about this one https://github.com/dotansimha/graphql-eslint/blob/new-options-for-require-description/docs/rules/require-description.md and https://github.com/dotansimha/graphql-eslint/blob/new-options-for-require-description/docs/rules/naming-convention.md
Also, I referenced the last spec of `October2021` instead of `June2018` :)

Reply:
Haha. Yes, definitely. Good catch. As I sent it, I was a little surprised that the spec hadn't been updated since June2018. 😅